packages/scalescss/index.html

```diff
@@ -2403,24 +2403,24 @@ <h4>Usage:</h4>
         </div>
       </section>
 
-      <section id="stack-split">
+      <section id="stack">
         <header>
-          <h2>Stack Split</h2>
+          <h2>Stack</h2>
         </header>
         <div class="pattern">
-          <h3>Default Stack Split</h3>
+          <h3>Stack</h3>
```

Comment:
I would call this "Default Stack" and just show what it does by itself. Then one section that's "Stack with Split Modifier" which just shows one with medium or whatever. And the take that same markup and do "Stack with Split Modifier - Tiny Spacing". One of those for each spacing section. I don't think we need every combination of split and size, so just default, one example of split, and then that same split example with each size modifier applied. That should be sufficient.